audio-worklet/basic/js/noise-generator.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+registerProcessor('noise-generator', class Noise extends AudioWorkletProcessor {
+  static get parameterDescriptors () {
+    return [{
+      name: 'gain',
+      defaultValue: 0.25,
```

Comment:
Why a default value of 0.25?  I would expect the default amplitude of a uniform RNG to be one.